pkgs/by-name/vl/vlc-bittorrent/package.nix

```diff
@@ -0,0 +1,66 @@
+{ autoconf
+, autoconf-archive
+, automake
+, boost
+, fetchFromGitHub
+, gcc
+, gnumake
+, lib
+, libtool
+, libtorrent-rasterbar
+, libvlc
+, openssl
+, pkg-config
+, stdenv
+}:
+
+# VLC does not know where the vlc-bittorrent package is installed.
+# make sure to have something like:
+#   environment.variables.VLC_PLUGIN_PATH = "${pkgs.vlc-bittorent}";
+
+stdenv.mkDerivation rec {
+  pname = "vlc-bittorrent";
+  version = "2.15.0";
+
+  src = fetchFromGitHub {
+    owner = "johang";
+    repo = pname;
+    rev = "6810d479e6c1f64046d3b30efe78774b49d1c95b";
+    hash = "sha256-7FHeQYHbMKZJ3yeHqxTTAUwghTje+gEX8gSEJzfG5sQ=";
+  };
+
+  nativeBuildInputs = [
+    gcc
+    autoconf
+    autoconf-archive
+    automake
+    gnumake
+    pkg-config
+  ];
+
+  buildInputs = [
+    libtorrent-rasterbar
+    libvlc
+    libtool
+    openssl
+    boost
+  ];
+
+  buildPhase = ''
+    autoreconf -i
+    ./configure --prefix=$out
+    make
+  '';
+
+  installPhase = ''
+    make install
+  '';
+
+  meta = with lib; {
+    description = "A bittorrent plugin for VLC.";
+    homepage = "https://github.com/johang/vlc-bittorrent";
+    license = licenses.gpl3;
```

Comment:
Should be `gpl3Only` or `gpl3Plus`.

Reply:
Just to confirm: `gpl3Plus` is for dual-licence?

Reply:
Yes, it's for GPL3 or later.

Reply:
Sorry, I am unsure how to tell. Is everything that uses GPLv3 - 29 June 2007 or later considered as `gpl3Plus`? Not very well-versed in legal terms.

Reply:
It should be mentioned in the license file, something like "or later" appears in the text.

Reply:
Thank you. It is indeed `gpl3Plus` then.